### DISCUSSION TITLE
How To Sync Firebase Realtime Database

### DISCUSSION THREAD

**Discussion by boyintheroom** [shashi278/social-auth-kivy#24] [Q&A]
Hi Shashi sir, very thankful you created this. one thing is bugging me lately.
if we log in with google how do we connect with firebase realtime database. suppose i have unique usernames to use it to differentiate users, when i log in i want to put the username and some data like age, class, weight, height to the realtime database how can i do that sir.

**Comment by shashi278** [comment]
@boyintheroom This project only helps you with social-logins. You might have to manage database on your own.
